terasoluna-gfw-parent/pom.xml

```diff
@@ -363,22 +363,6 @@
       </dependency>
       <!-- == End Joda-Time == -->
 
-      <!-- == Begin Logging == -->
-      <!-- Log4JDBC -->
-      <dependency>
-        <groupId>org.lazyluke</groupId>
-        <artifactId>log4jdbc-remix</artifactId>
-        <version>${org.lazyluke.version}</version>
-        <!-- this artifact depends on junit as compile scope -->
```

Comment:
- [ ] below property should remove .

https://github.com/terasolunaorg/terasoluna-gfw/blob/6449d0ecc36e353b2c81e60926601e7ead032102/terasoluna-gfw-parent/pom.xml#L536

Reply:
The target property has been deleted.

0d220bfb1a009226003aa7b7ef92ef26cc308afe